app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -63,20 +84,33 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     * @param lat2
     * @param lng2
     * @param severity
+    * @param filetype
     * @return
     */
-  def getAccessAttributesWithLabelsV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String]) = UserAwareAction.async { implicit request =>
+  def getAccessAttributesWithLabelsV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String], filetype: Option[String]) = UserAwareAction.async { implicit request =>
     apiLogging(request.remoteAddress, request.identity, request.toString)
 
     val minLat:Float = min(lat1, lat2).toFloat
     val maxLat:Float = max(lat1, lat2).toFloat
     val minLng:Float = min(lng1, lng2).toFloat
     val maxLng:Float = max(lng1, lng2).toFloat
 
-    val features: List[JsObject] =
-      GlobalAttributeTable.getGlobalAttributesWithLabelsInBoundingBox(minLat, minLng, maxLat, maxLng, severity).map(_.toJSON)
-
-    Future.successful(Ok(Json.obj("type" -> "FeatureCollection", "features" -> features)))
+    if (filetype != None && filetype.get == "csv") {  // in CSV format
+      val file = new java.io.File("access_attributes_with_labels.csv")
+      val writer = new java.io.PrintStream(file)
+      writer.print("Attribute ID,Label Type,Attribute Severity,Attribute Temporary,Neighborhood Name,Label ID,")
```

Comment:
I think this should be a single println statement. You can split a string across multiple lines so it doesn't go past 120 characters.

Reply:
This applies for the other functions as well.

Reply:
the println() function didn’t let me split lines with a string literal input, should I just make a string value with the long string and then pass it in?

Reply:
Yeah that would be best